tests/integration/api_v3/rpc/posStakes.test.js

```diff
@@ -19,68 +19,63 @@ const { request } = require('../../../helpers/socketIoRpcRequest');
 const {
 	invalidParamsSchema,
 	jsonRpcEnvelopeSchema,
-	metaSchema,
 } = require('../../../schemas/rpcGenerics.schema');
 
 const {
-	voteSchema,
-} = require('../../../schemas/api_v3/vote.schema');
+	stakeResponseSchema,
+} = require('../../../schemas/api_v3/stake.schema');
 
 const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
 
-const getVotes = async (params) => request(wsRpcUrl, 'get.dpos.votes.sent', params);
+const getVotes = async (params) => request(wsRpcUrl, 'get.pos.stakes', params);
 
-describe('get.dpos.votes.sent', () => {
-	let refDelegate;
+describe('get.pos.stakes', () => {
+	let refValidator;
 	beforeAll(async () => {
 		let response;
 		do {
 			// eslint-disable-next-line no-await-in-loop
 			response = await request(wsRpcUrl, 'get.dpos.delegates', { limit: 1 });
 		} while (!response.result);
-		[refDelegate] = response.result.data;
+		[refValidator] = response.result.data;
 	});
 
 	it('Returns list of votes when requested for existing account by address', async () => {
```

Comment:
Same for RPC tests as well
```suggestion
	it('Returns list of stakes when requested for existing account by address', async () => {
```